src/ram.asm

```diff
@@ -8,14 +8,23 @@ tmpZ: .res 1 ;  $0005
 tmpBulkCopyToPpuReturnAddr: .res 2 ;  $0006 ; 2 bytes
 binScore: .res 4 ;  $8 ; 4 bytes binary
 score: .res 4 ;  $C ; 4 bytes BCD
-    .res 7
+nmiReturnAddr: .res 1 ; $0010 ; used for crash
```

Comment:
no, it doesn't need to be in the zeropage. you can move things if you want, but it would take up more PRG due to longer addresses. allegroIndex is referenced the most so that should probably be kept on zeropage if anything.